flask_caching/backends/filesystem.py

```diff
@@ -14,14 +14,13 @@
 import os
 import tempfile
 from time import time
-
 from werkzeug.posixemulation import rename
 
 from flask_caching.backends.base import BaseCache
 
 try:
-    import cPickle as pickle
-except ImportError:  # pragma: no cover
+    import _pickle as pickle
```

Comment:
In Python 3 it is advisable to import `pickle` instead. It will import and use `cPickle` if available.